src/components/Articles/Article/Article.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import 'dotenv/config';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import LazyLoad from 'react-lazyload';
+import Header from '../../common/Header/Header';
+import { fetchOneArticle } from '../../../actions';
+import avatar from '../../../assets/images/user.png';
+// import TimeStamp from '../../../helpers/TimeStamp';
+import './Article.scss';
+
+const { REACT_APP_IMAGE_BASE_URL } = process.env;
```

Comment:
This should be placed in a helper file not directly here.

Reply:
@luc-tuyishime, are you referring to the `REACT_APP_IMAGE_BASE_URL`?